Note on transformation of static variables by attribute exception (#251)
rust-embedded/book: src/start/exceptions.md

`exception` handlers can *not* be called by software thus reentrancy is not
possible.

> > Note that the `exception` attribute makes transformation of static variables

Reviewer: This sentence no English. 😅

Author: @therealprof I have changed wording. Please, take a look

> inside the function by wrapping them into `unsafe` blocks and providing us

> > with new appropriate variables of type `&mut` which names match user defined.

Reviewer: ```suggestion
> with new appropriate variables of type `&mut` of the same name.
```

> > Thus we can use operator `*` to access the values of the variables without

Reviewer: ```suggestion
> Thus we can derefence the reference via `*` to access the values of the variables without
```

> > need to wrap them into the `unsafe` blocks.

Reviewer: ```suggestion
> needing to wrap them in an `unsafe` block.
```
